### ISSUE TITLE
Data Source Refresh Button

### ISSUE THREAD

**Issue by terrypacker** [infiniteautomation/ma-dashboards#127]
As per this request, when creating the new Data Source UI consider the following:

https://github.com/infiniteautomation/ma-core-public/issues/397

**Comment by jazdw** [comment]
@terrypacker is there an end point for this?

**Comment by terrypacker** [comment]
@jazdw no.  But there is a method in the data source service so this would be easy to add.  DataSourceService.forceDataSourcePoll(id);  We should probably add another method that accepts an XID and expose this on an endpoint.
